sklearn/ensemble/_forest.py

```diff
@@ -1110,10 +1110,11 @@ class RandomForestClassifier(ForestClassifier):
            The default value of ``n_estimators`` changed from 10 to 100
            in 0.22.
 
-    criterion : {"gini", "entropy", "log_loss"}, default="gini"
+    criterion : {"gini", "entropy", "log_loss", "hellinger"}, default="gini"
         The function to measure the quality of a split. Supported criteria are
-        "gini" for the Gini impurity and "log_loss" and "entropy" both for the
-        Shannon information gain, see :ref:`tree_mathematical_formulation`.
+        "gini" for the Gini impurity, "log_loss" and "entropy" both for the
+        Shannon information gain, see :ref:`tree_mathematical_formulation`
+        and "hellinger" for the Hellinger distance.
```

Comment:
Similarly, I think a list to describe each criterion with the reference to the mathematical section after it would be more readable.

This suggestions also applies to other docstrings.

Reply:
done